src/sbvr-api/permissions.ts

```diff
@@ -715,19 +715,18 @@ const onceGetter = <T, U extends keyof T>(
 };
 
 const deepFreezeExceptDefinition = (obj: AnyObject) => {
```

Comment:
`ExceptDefinition` isn't accurate to what this is attempting to do

Reply:
Yep corrected.